script/install.sh

```diff
@@ -147,17 +147,18 @@ Options
   -v --verbose  Verbose output
 
 Run "install.sh <command> --help to get help for a specific command.
+If no argument are provided, the latest DMD compiler will be installed.
 '
 }
 
 command_help() {
     local _compiler='Compiler
 
   dmd|gdc|ldc           latest version of a compiler
-  dmd|gdc|ldc-<version> specific version of a compiler (e.g. dmd-2.069.0, ldc-0.16.1-beta2)
+  dmd|gdc|ldc-<version> specific version of a compiler (e.g. dmd-2.071.1, ldc-1.1.0-beta2)
```

Comment:
I thought it's nice if the examples are a bit up-to-date, but I can revert that if you see it as noise.